### DISCUSSION TITLE
Unable to login

### DISCUSSION THREAD

**Discussion by hiepvq** [vercel/community#4309] [Account Issue] [answered]
### Question

i created the account using my email, i verified my account but still can not login
pls you help to fix it
![Screenshot 2023-10-01 173917](https://github.com/vercel/community/assets/36187674/ade306e9-ee57-46b2-9a20-1eb1a348a940)

**Comment by amyegan** [comment, marked as answer]
Sorry you're having trouble with your account. We can't safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps. If so, please follow those steps.

To open a ticket, please use the appropriate [direct channel](https://vercel.com/guides/how-to-get-vercel-support#direct-channels) found in this guide: [What is the best way to get support from Vercel?](https://vercel.com/guides/how-to-get-vercel-support#direct-channels)
